ci_framework/hooks/templates/service_config_ceph.yaml.j2

```diff
@@ -0,0 +1,71 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ./{{ cifmw_kustomize_cr_file_name }}
+namespace: {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
+patches:
+- patch: |-
+    - op: add
+      path: /spec/extraMounts
+      value:
+        - name: v1
+          region: r1
+          extraVol:
+          - propagation:
+            - Glance
+            - volume1
+            - CinderBackup
+            extraVolType: Ceph
+            volumes:
+            - name: ceph
+              projected:
+                sources:
+                - secret:
+                    name: ceph-conf-files
+            mounts:
+            - name: ceph
+              mountPath: "/etc/ceph"
+              readOnly: true
+    - op: replace
+      path: /spec/cinder/template/cinderVolumes/volume1/replicas
+      value: 1
+    - op: add
+      path: /spec/cinder/template/cinderVolumes/volume1/customServiceConfig
+      value: |
+            [DEFAULT]
+            enabled_backends=ceph
+            debug=True
+            [ceph]
+            volume_backend_name=ceph
+            volume_driver=cinder.volume.drivers.rbd.RBDDriver
+            rbd_ceph_conf=/etc/ceph/ceph.conf
+            rbd_user=openstack
+            rbd_pool=volumes
+            rbd_flatten_volume_from_snapshot=False
+            report_discard_supported=True
+            backend_host=hostgroup
+            rbd_secret_uuid={{ cifmw_ceph_fsid }}
+    - op: replace
+      path: /spec/cinder/template/cinderBackup/replicas
+      value: 1
+    - op: add
+      path: /spec/cinder/template/cinderBackup/customServiceConfig
+      value: |
+            [DEFAULT]
+            backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
+            backup_ceph_pool = backups
+            backup_ceph_user = openstack
+    - op: add
+      path: /spec/glance/template/customServiceConfig
+      value: |
+            [DEFAULT]
+            debug = true
+            enabled_backends=default_backend:rbd
+            [glance_store]
+            default_backend=default_backend
+            [default_backend]
+            rbd_store_ceph_conf=/etc/ceph/ceph.conf
+            rbd_store_user=openstack
+            rbd_store_pool=images
```

Comment:
These values are already present in https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml#L49 Do we still need to kustomize it?